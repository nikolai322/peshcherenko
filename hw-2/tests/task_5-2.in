3
1 -1 1
4
1.0 -0.25 0.75 0.0
