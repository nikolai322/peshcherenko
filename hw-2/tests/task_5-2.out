1
1.500000
